code/modules/admin/verbs/randomverbs.dm

```diff
@@ -650,7 +650,7 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	var/flash = input("Range of flash. -1 to none", text("Input"))  as num|null
 	if(flash == null) return
 	var/shaped = 0
-	if(config.use_recursive_explosions)
+	if(config.use_iterative_explosions)
```

Comment:
Should probably disable this for iterative unless you're going to add shaping support to it.